ql/src/test/queries/clientpositive/jdbc_table_dml_mysql.q

```diff
@@ -0,0 +1,15 @@
+--! qt:database:mysql:q_test_country_table.sql
+
+CREATE EXTERNAL TABLE country (id int, name varchar(20))
+    STORED BY 'org.apache.hive.storage.jdbc.JdbcStorageHandler'
+TBLPROPERTIES (
+    "hive.sql.database.type" = "MYSQL",
+    "hive.sql.jdbc.driver" = "com.mysql.jdbc.Driver",
+    "hive.sql.jdbc.url" = "jdbc:mysql://localhost:3306/qtestDB",
+    "hive.sql.dbcp.username" = "root",
+    "hive.sql.dbcp.password" = "qtestpassword",
+    "hive.sql.table" = "country"
+    );
+
+INSERT INTO country VALUES (8, 'Hungary');
+SELECT * FROM country;
```

Comment:
Please add new line at the end of file (see https://unix.stackexchange.com/questions/18743/whats-the-point-in-adding-a-new-line-to-the-end-of-a-file).

Reply:
Done.